### PR TITLE
Add operators.json for C++. Resolves #298

### LINKED ISSUE #298: Add C++ Logic/Math Operators
# Description

Logic/Math Operators for C++ aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Requirements

* Copy `web/thesauruses/_meta/operators.json` to the right C++ directory in `web/thesauruses/`
* Edit the operators.json file to accurately reflect the latest major version of C++
* Delete the `"name":` fields before your PR
* Add in `"comment":` if you wish to explain something in the code
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then leave `"code":` blank.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

### PR DESCRIPTION
**Pull request by Ranjana-Kambhammettu**
## What GitHub issue does this PR apply to?

Resolves #298 

## What changed and why?

Added operators.json file and added the code for all the relevant operators in C++.

## Any additional comments or things to be aware of while reviewing?

None. I have tested this according to the documentation on my system.

## Reviewer Steps

- [x] Review commits
- [x] (If needed) Approve first run of workflow
- [ ] (If needed) Activate Heroku Review app
- [ ] Verify any changes in Deploy Preview
- [ ] Approve and/or merge changes

### CODE REVIEW

File: web/thesauruses/cpp/operators.json [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    },
+    "greater_than": {
+      "code": ">"
+    },
+    "greater_than_or_equal_to": {
+      "code": ">="
+    },
+    "null_coalescing": {
+      "not-implemented": "true",
+      "code": ""
```
Comment (geekygirlsarah) [inline web/thesauruses/cpp/operators.json]: For the `not-implemented` ones, could you remove the `code` lines?

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]

**Comment by geekygirlsarah** [comment]
I think this is good if you remove the code blocks from the not-implemented parts. Then I'll merge it in!

File: web/thesauruses/cpp/operators.json (hunk 1 above)
Comment (Ranjana-Kambhammettu) [reply web/thesauruses/cpp/operators.json]: I have made the necessary changes 👍🏽

**Comment by geekygirlsarah** [comment]
The JSON Check failed because the JSON file isn't valid. Looks like every line with `not-implemented: true` has trailing commas at the end of each line, which isn't valid. If you remove those and the check passes, I can merge it in.

File: web/thesauruses/cpp/operators.json [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    "increment": {
+      "comment": "There are two types of increment: pre-increment (++x) and post-increment (x++). ",
+      "code": "++"
+    },
+    "decrement": {
+      "comment": "There are two types of decrement: pre-decrement (--x) and post-decrement (x--). ",
+      "code": "--"
+    },
+    "exponential": {
+      "not-implemented": "true",
```
Comment (geekygirlsarah) [inline web/thesauruses/cpp/operators.json]: Lines that seem to have `not-implemented` have trailing commas, which aren't valid JSON.

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]

File: web/thesauruses/cpp/operators.json (hunk 2 above)
Comment (Ranjana-Kambhammettu) [reply web/thesauruses/cpp/operators.json]: I have made the necessary changes.

**Comment by geekygirlsarah** [comment]
Alright, this passed the checks! I'll merge it in. Thank you!

Don't forget this counts for the Hacktoberfest shirt if you're signed up.

**Pull request merged**